dbflute_maihamadb/freegen/remoteapi/ControlFreeGenRemoteApiJava.vm

```diff
@@ -13,46 +13,19 @@
 ## either express or implied. See the License for the specific language
 ## governing permissions and limitations under the License.
 ##
+##########################################################################################
+##### Ready for generation.
+##########################################################################################
+##### Preparing to execute javascript.
 #set ($scriptEngine = $manager.createJavaScriptEngine())
 $scriptEngine.put('scriptEngine', $scriptEngine)
+##
+##### Set to use FreeGen objects when running javascirpt.
 $scriptEngine.put('generator', $generator)
 $scriptEngine.put('manager', $manager)
+##
+##########################################################################################
+##### Generation process.
+##########################################################################################
 $scriptEngine.eval('load("./freegen/remoteapi/ControlFreeGenRemoteApiJava.js");')
 $scriptEngine.invokeFunction('process', $requestList)
```

Comment:
これ、ScriptEngineインターフェースじゃなくて、実はInvocableインターフェースのメソッドなんだよね。
NashornScriptEngineもRhinoScriptEngineもimplementsしてるので、ScripgEngineだったら大体implementsするものなんだろうけど...
一応、一言「ScriptEngineがInvocableインターフェースをimplementsしてることが前提」みたいなことを書いておいてもらえれば。
(この行じゃなくて、$scriptEngineの宣言のところでも良いかもだけど)

Reply:
$scriptEngineは、ScriptEngine、Invocableをimplementsしているクラスを想定
にしようと思います。

NashornScriptEngine、RhinoScriptEngineはCompilableもimplementsしていますが、現状RemoteApiGenでは使っていないので。

Reply:
対応しました。

Reply:
想定了解。したらコメントよろしく。

Reply:
おっと、もうコメントもすでに書いたよって話なのね？ありがとう。
```
##### but since it will disappear after Java 15, there is a possibility that it is an alternative engine such as sai.
##### $scriptEngine must be a class that implements ScriptEngine, Invocable.
```

※そっちでコメント書きましたって一言言ってくれると嬉しいかな（＾＾。